docs/content/docs/test/Runner.md

```diff
@@ -143,7 +143,7 @@ In our case:
 
 Check out a configuration to run in `GradleInstrumentationPluginConfiguration` in the [test app](https://github.com/avito-tech/avito-android/blob/develop/samples/test-app/build.gradle.kts#L114)
 
-## Filtering tests for execution
+## Filtering tests from execution
```

Comment:
Are you sure?
We filter tests before execution. Execution is the purpose of filtering. 
https://wikidiff.com/content/vs-whats-difference-33

Reply:
No

Reply:
```suggestion
## Filtering tests for execution
```